resources/views/emails/edit-comment.blade.php

```diff
@@ -22,3 +22,7 @@
         </td>
     </tr>
 </table>
+
+@if($unsubscribeLink)
```

Comment:
Is there a context in which there won't be an unsubscribe link?

Also, we prefer spaces after Blade control structures, so `@if ($unsubscribeLink)`

Reply:
I can't think of an instance where the link wouldn't exist, but defensively didn't want to have an empty `href` attribute if it was sent to the view as `null`.